sdk/spring/spring-cloud-azure-starter-monitor-test/src/test/java/com/azure/SpringMonitorTest.java

```diff
@@ -68,6 +69,8 @@ SelfDiagnosticsLevel testSelfDiagnosticsLevel() {
         }
     }
 
+    // TODO - this test is disabled because it requires spring starter to be migrated to azure-json
+    @Disabled
     @Test
     public void shouldMonitor() throws InterruptedException, MalformedURLException {
```

Comment:
I have run this test sucessfully with your branch and IntelliJ, but IntelliJ may laught at me. 

It's surprising that the changes of this PR lead to the failure of this test, because the project of this test uses `spring-cloud-azure-starter-monitor` based on an OTel exporter version released to Maven central: https://github.com/heyams/azure-sdk-for-java/blob/0868a0afcec9cc07dfa70faaa9b1aa2031e13b2a/sdk/spring/spring-cloud-azure-starter-monitor/pom.xml#L101

Could you provide an error message?

Reply:
i ran it locally today and it was successful. Will check the CI build.